mw-webapp/src/model/businessModel/Comment.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * User's comments model
+ */
+export class Comment {
+
+  /**
+   * Comment's UUID
+   */
+  public uuid: string;
+
+  /**
+   * Owner's uui
+   */
+  public ownerUuid: string;
```

Comment:
we use owner name for user who own the way, why should we use the same name for commentator?

Reply:
Chaned